src/Integration.Vsix/Helpers/IStatusBarNotifier.cs

```diff
@@ -36,25 +38,28 @@ internal interface IStatusBarNotifier
     internal class StatusBarNotifier : IStatusBarNotifier
     {
         private IVsStatusbar vsStatusBar;
+        private readonly IThreadHandling threadHandling;
 
         [ImportingConstructor]
-        public StatusBarNotifier([Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider)
+        public StatusBarNotifier([Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider, IThreadHandling threadHandling)
         {
-            RunOnUIThread.Run(() =>
+            this.threadHandling = threadHandling;
+
+            threadHandling.RunOnUIThread(() =>
             {
                 vsStatusBar = serviceProvider.GetService(typeof(IVsStatusbar)) as IVsStatusbar;
-            });
+            }).Forget();
         }
 
         public void Notify(string message, bool showSpinner)
```

Comment:
I think we should open a ticket to change this method to be async so that the caller could decide if to call Forget or await.